closed/autoconf/custom-hook.m4

```diff
@@ -198,7 +234,12 @@ AC_DEFUN_ONCE([OPENJ9_PLATFORM_SETUP],
 AC_DEFUN_ONCE([OPENJDK_VERSION_DETAILS],
 [
   OPENJDK_SHA=`git -C $TOPDIR rev-parse --short HEAD`
-  OPENJDK_TAG=`git -C $TOPDIR describe --abbrev=0 --tags --match "jdk-10*" "${OPENJDK_SHA}"`
+  LAST_TAGGED_SHA=`git -C $TOPDIR rev-list --tags="jdk-10*" --max-count=1 2>/dev/null`
+  if test "x$LAST_TAGGED_SHA" != x; then
+    OPENJDK_TAG=`git -C $TOPDIR describe --abbrev=0 --tags "$LAST_TAGGED_SHA"`
```

Comment:
The option `--abbrev=0` seems at best redundant here. Since we're always referring to a commit which is tagged, there's nothing to abbreviate.